### PR TITLE
Run in loops

### PR DESCRIPTION
**Pull request by ItaiMaoz**

### CODE REVIEW

**Comment by ItaiMaoz** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: english/Computer-Man/Computer-Man.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 You're walking down the street, and the glint of a flashy new computer
 catches your eye through the window.  You walk into the store, and
-meet.......the [computer man](https://www.youtube.com/watch?v=Hm4E8TcjF70).
\ No newline at end of file
+meet.......the [computer man](https://www.youtube.com/watch?v=Hm4E8TcjF70).
+
+[Restart] (../../Marshmallow.md)
```
Comment (ItaiMaoz) [inline english/Computer-Man/Computer-Man.md]: @casey-collab , I believe I fixed the issue. please recheck. thanks.

**Comment by ItaiMaoz** [comment]
I've resolved the link. Please recheck. thanks.

**Comment by ItaiMaoz** [comment]
@casey-collab, please review

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**